### PR TITLE
2nd set of python 2 and 3 compatible changes

### PR DESCRIPTION
**Pull request by razasayed**
This PR contains the following changes:

1. Using absolute imports

2. Changes to attach a metaclass

3. Fixing the filter method

4. Using six.moves.urllib and six.text_type

### CODE REVIEW

File: language_utilities/utils.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 from language_utilities.constant import ENGLISH_LANG
 from language_utilities.constant import TRANSLATED_TEXT
 from chatbot_ner.config import ner_logger, GOOGLE_TRANSLATE_API_KEY
-import urllib
+import six.moves.urllib.request, six.moves.urllib.parse, six.moves.urllib.error
```
Comment (srajdev) [inline language_utilities/utils.py]: E401 multiple imports on one line

File: language_utilities/utils.py [hunk 2]
```diff
@@ -18,8 +18,8 @@ def unicode_urlencode(params):
         (str): url string with params encoded
     """
     if isinstance(params, dict):
-        params = params.items()
-    return urllib.urlencode([(k, isinstance(v, unicode) and v.encode('utf-8') or v) for k, v in params])
+        params = list(params.items())
+    return six.moves.urllib.parse.urlencode([(k, isinstance(v, six.text_type) and v.encode('utf-8') or v) for k, v in params])
```
Comment (srajdev) [inline language_utilities/utils.py]: E501 line too long (126 > 119 characters)

File: models/crf_v2/get_crf_tagger.py [hunk 3]
```diff
@@ -1,15 +1,16 @@
...
 
 
-class CrfModel(object):
+class CrfModel(six.with_metaclass(Singleton, object)):
     """
     This class is used to load the crf tagger for a given entity
     """
-    __metaclass__ = Singleton
-
+
```
Comment (srajdev) [inline models/crf_v2/get_crf_tagger.py]: W293 blank line contains whitespace

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

File: language_utilities/utils.py [hunk 4]
```diff
@@ -18,8 +20,11 @@ def unicode_urlencode(params):
         (str): url string with params encoded
     """
     if isinstance(params, dict):
-        params = params.items()
-    return urllib.urlencode([(k, isinstance(v, unicode) and v.encode('utf-8') or v) for k, v in params])
+        params = list(params.items())
+    return six.moves.urllib.parse.urlencode(
+        [(k,
+          isinstance(v, six.text_type) and v.encode('utf-8') or v) for k, v in params
+        ])
```
Comment (srajdev) [inline language_utilities/utils.py]: E124 closing bracket does not match visual indentation

File: language_utilities/utils.py [hunk 5]
```diff
@@ -18,8 +20,12 @@ def unicode_urlencode(params):
         (str): url string with params encoded
     """
     if isinstance(params, dict):
-        params = params.items()
-    return urllib.urlencode([(k, isinstance(v, unicode) and v.encode('utf-8') or v) for k, v in params])
+        params = list(params.items())
+    return six.moves.urllib.parse.urlencode(
+        [(k,
+          isinstance(v, six.text_type) and v.encode('utf-8') or v) for k, v in params
+        ]
```
Comment (srajdev) [inline language_utilities/utils.py]: E124 closing bracket does not match visual indentation

File: language_utilities/utils.py [hunk 6]
```diff
@@ -18,8 +20,10 @@ def unicode_urlencode(params):
         (str): url string with params encoded
     """
     if isinstance(params, dict):
-        params = params.items()
-    return urllib.urlencode([(k, isinstance(v, unicode) and v.encode('utf-8') or v) for k, v in params])
+        params = list(params.items())
+    return six.moves.urllib.parse.urlencode([(k, isinstance(v,
+        six.text_type) and v.encode('utf-8') or v) for (k, v) in
```
Comment (srajdev) [inline language_utilities/utils.py]: E128 continuation line under-indented for visual indent

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by razasayed** [comment]
@ashutoshsingh0223 @chiragjn lint errors fixed

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

**Pull request merged**